<launch>
    <node name="wifi_publisher" pkg="wifi_localization" type="wifi_publisher.py" output="screen">
        <param name="wifi_interface" type="string" value="wlan0" />
    </node>
</launch>
